src/components/Sidebar/styles.ts

```diff
@@ -0,0 +1,29 @@
+
+import { makeStyles } from '@mui/styles';
+
+export default makeStyles(() => ({
+  sideBar: {
+    height: '100vh',
+    backgroundColor: '#3440e3',
+  },
+  imageLink: {
+    display: 'flex',
+    justifyContent: 'center',
+    padding: '10% 0',
```

Comment:
Tip: using percentage padding causes your layout to be inconsistent if the viewport height changes (e.g. phone rotating from portrait to landscape)

Reply:
Noted - I have changed it to **rem** so that it can pick the room measurement to work with.